application/views/userManagement/partial/confirmuseractivation.php

```diff
@@ -0,0 +1,22 @@
+<?php
+Yii::app()->getController()->renderPartial(
+    '/layouts/partial_modals/modal_header',
+    ['modalTitle' => gT(ucfirst($action) .' user')]
+);
+?>
+
+<div class="modal-body">
+    <p class='modal-body-text'><?php eT("Do you want to $action this user?"); ?></p>
```

Comment:
Not a valid localized string. :) Please see https://manual.limesurvey.org/Code_quality_guide#Localization